README.md

```diff
@@ -16,25 +16,39 @@ Or
 
 
 ## Run with Docker
-
+- If you are on your local machine, we recommend that you run this container using Docker.  
+- We have tested running this container using Docker on local mac machines and also on a Pawsey Nimbus instance.
+  
+### On a local (mac) machine
```

Comment:
Split this out into 2 subsections: 

```
### On a local machine 

#### MacOS 

#### Windows

Reply:
For windows you will need to suggest users run docker using a linux terminal emulator like MobaXterm

Reply:
Relavant comments added